db/version_set.cc

```diff
@@ -3811,7 +3811,7 @@ Status VersionSet::ProcessManifestWrites(
                builder_guards.size() == versions.size());
         ColumnFamilyData* cfd = versions[i]->cfd_;
         s = builder_guards[i]->version_builder()->LoadTableHandlers(
-            cfd->internal_stats(), cfd->ioptions()->optimize_filters_for_hits,
+            cfd->internal_stats(), 1 /* max_threads */,
```

Comment:
This one doesn't really feel like a user-visible bug. No matter the value of `cfd->ioptions()->optimize_filters_for_hits` in the old code (0 or 1), it shouldn't have created any additional threads for file opening.

Reply:
Hmm... Maybe you are right. I looked at https://en.cppreference.com/w/cpp/language/implicit_conversion and it does seem that bool is can only be implicitly converted to 0 and 1. Let me update the description.